frameworks/Python/aiohttp/aiohttp-gunicorn.dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM python:3.13
+FROM python:latest
```

Comment:
`FROM python:3` it's better.
If python use SEM versioning is not problem.
But `latest` is not so !!!
If you will be here for some years, and fix the problems it's OK, but now this line create a problem.
Thank you !!

Reply:
Suppose that makes sense.

Reply:
Actually, I'm not sure it'll make any difference. Any changes for a v4 are no more likely to break the small amount of code here than a minor release would. Ideally we'd be able to have Dependabot bump the pinned version automatically...

Reply:
I think I'm okay with going with `latest` and if it breaks something we'll see it pretty plainly.

Reply:
Actually, thinking on this for more than one second, it would make it harder for people to understand what version was run on any given continuous benchmark run from day to day. Imagine some breaking change from 3.13 to 3.14 (as a contrived example) but this dockerfile says `latest` - might be hard without some guesswork what's going on. Agreeing with @joanhey that this should be returned to sem. ver.